scripts/_generate/index.js

```diff
@@ -200,42 +203,59 @@ function generateScripts () {
                     exclude: /(node_modules|bower_components)/,
                     use: [
                         'babel-loader',
-                        'eslint-loader',
+                        {
+                            loader: 'eslint-loader',
+                            options: {
+                                globals: [
+                                    'HelixUI',
+                                ],
+                            },
+                        },
```

Comment:
Add `HelixUI` as a known global var so that eslint doesn't complain when we manually initialize in `docs/docs.js` via `HelixUI.initialize()`.